PySDM/physics/drop_growth/howell_1949.py

```diff
@@ -14,7 +14,17 @@ class Howell1949(Fick):  # pylint: disable=too-few-public-methods
 
     @staticmethod
     def Fk(const, T, K, lv):
-        """thermodynamic term associated with heat conduction"""
+        """Thermodynamic term associated with heat conduction.
+
+        Parameters
+        ----------
+        T
+            Temperature in Kelvins.
```

Comment:
since the other parameters do not have units defined in the docstring, and we embrace only-SI unit policy across the codebase, it is likely unneeded to add "in Kelvins" here